### DISCUSSION TITLE
Сожжение тумбочки Фила: способы

### DISCUSSION THREAD

**Discussion by sntxerror** [b4ck5p4c3/0x08.in#358] [Ideas] [answered]
Я думаю могу сжечь своим взглядом из стикерпака после парочки горячих обсуждений в чате

**Comment by glitchcore** [comment]
Так, чо тут в дискашшенах, все как в ишью или есть фишки?

**Reply by sntxerror** [reply]
есть ветки например

**Reply by glitchcore** [reply]
А, заебись, можно отвечать на ответы

**Reply by sntxerror** [reply]
еще можно голосовать за темы

**Reply by glitchcore** [reply]
Но не больше 1 уровня вложенности, да?

**Reply by sntxerror** [reply]
> 
> 
> А, заебись, можно отвечать на ответы

но ветка с одним уровнем вложенности БЛЕАН

**Reply by sntxerror** [reply]
> 
> 
> Но не больше 1 уровня вложенности, да?

лучше чем ничего, мне кажется

**Comment by lomalkin** [comment]
Чем issue отличаются от дискуссий, кто-то может пояснить?

**Reply by glitchcore** [reply]
Тут есть ветки, мы выше обсуждали

**Reply by sntxerror** [reply]
issue относятся к артефактам в репозитории, дискачи нет

**Comment by glitchcore** [comment, marked as answer]
Принято решение тумбочку не сжигать, ограничиться мэппингом.
